free-3.md

```diff
@@ -10,34 +11,49 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+The project would make automatic trolling on social media more accessible to a wider audience. It would allow anyone to create a bot on social media websites and program it with simple functionality. A language would be a good way to meet these needs because the UI could be accessible to less technologically capable users.
+
+The idea seeks to help those who want to create [bots](https://twitter.com/phasechase) on social media but might not have the technical expertise to do so. People could currently have clever (or incredibly annoying) ideas for bots but lack the knowledge to make them. If a language could help them by making the creation of bots more accessible, they could make their ideas into reality.
+
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+The primary benefit of a DSL in this situation would be to simplify bot creation. It addresses the need for a tool that is usable by people who do not have programming knowledge. The DSL would basically be a graphical overlay for bot creation APIs.
+
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+I have tried to create bots for social media in the past and ran into various errors throughout the process. Even as someone with a computer science background, it is difficult to get a bot running. I have seen a wide variety of novelty bots ranging from clever to annoying. My idea came from the desire to see more novelty bots without all of the pain involved in creating a custom bot.
```

Comment:
I know nothing about bot creation, but I imagine it's pretty difficult. Do you think it'll be easy to capture the necessary parts of bot making in a DSL that's easy to understand?